meilisearch-release-changelog/v0.30.0.md

```diff
@@ -0,0 +1,229 @@
+# v0.30.0 release changelogs
+
+<!-- The following line should NOT be put in the official release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+This release removes index deployment downtime and adds new task management features. It also introduces new search parameters that facilitate the creation of numbered pagination interfaces.
+
+Thanks to Hacktoberfest, tons of updates and bug fixes have been done by the fantastic Meilisearch community. You were more present than ever and words cannot make justice tojust  how grateful we are for your support. A big thank you from the whole Meilisearch team! ❤️
+
+<!-- The following line should NOT be put in the official release changelogs -->
+📖 Meilisearch's official documentation will be ready for the official v0.30.0 release. Meanwhile, work-in-progress documentation is available [here]().
+
+# New features and improvements 🔥
+
+## Ease search results navigation with pagination
+
+You can now navigate through page selection when searching with Meilisearch using the API. This way, you will get the exact total number of pages and results Meilisearch has matched based on a query.
+
+For this, use `page` and `hitsPerPage` as query parameter instead of `offset` and `limit`:
```

Comment:
```suggestion
You can now use `page` and `hitsPerPage` to fetch the total number of pages and results for a query. These new search parameters can be used in the place of `offset` and `limit` when creating interfaces that require knowing the exhaustive number of search results.

The following example fetches the second page of results for a given query:
```

Reply:
I would like the keep the pagination or navigation idea, WDYT? Same @gmourier WDYT?

Reply:
Both proposals seem relevant to me. I would add first that it is possible to request a specific result page for a search query.

Reply:
yeah, you're right, @curquiza. Edited my suggestion so we directly mention pagination.